l1-contracts/test/foundry/unit/concrete/Executor/ExecutorProof.t.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {ExecutorProvingTest} from "solpp/dev-contracts/test/ExecutorProvingTest.sol";
+import {IExecutor} from "solpp/zksync/interfaces/IExecutor.sol";
+import {VerifierParams} from "solpp/zksync/Storage.sol";
+
+contract ExecutorProofTest is Test {
+    ExecutorProvingTest internal executor;
+
+    function setUp() public {
+        executor = new ExecutorProvingTest();
```

Comment:
Idk whether you need this mock, maybe better to inherit `ExecutorFacet` contract and remove legacy dev contract